src/com/pentalog/pentastagiu/homework2/shop/ClientCart.java

```diff
@@ -0,0 +1,25 @@
+package com.pentalog.pentastagiu.homework2.shop;
+//
+public class ClientCart {
+
+    private ClientsOfShop name;
```

Comment:
name nu sugereaza ca ai o referinta de tip ClientsOfShop/(Client); de cele mai multe ori vom denumi fieldul cu acelasi nume ca cel al clasei, va reflecta f bine realitatea, in cazul tau: private Client client;